### PR TITLE
feat(DB): release ACDB 5.0.0

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Changes Proposed:
- release ACDB 5.0.0

## Tested:
image

### CODE REVIEW

**Comment by Winfidonarleyan** [comment]
@Kitzunu can fast check?

**Comment by Winfidonarleyan** [comment]
Ready

File: data/sql/base/db_characters/arena_team_member.sql [hunk 1]
```diff
@@ -21,11 +21,11 @@ CREATE TABLE IF NOT EXISTS `arena_team_member` (
   `weekWins` SMALLINT unsigned NOT NULL DEFAULT 0,
   `seasonGames` SMALLINT unsigned NOT NULL DEFAULT 0,
   `seasonWins` SMALLINT unsigned NOT NULL DEFAULT 0,
-  `personalRating` SMALLINT NOT NULL DEFAULT 0,
+  `personalRating` smallint(6) NOT NULL DEFAULT 0,
```
Comment (Kitzunu) [inline data/sql/base/db_characters/arena_team_member.sql]: No no

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Fix it everywhere

**Comment by Winfidonarleyan** [comment]
> 

Okay

**Comment by Winfidonarleyan** [comment]
> Fix it everywhere

Okay

File: data/sql/base/db_characters/arena_team_member.sql (hunk 1 above)
Comment (Winfidonarleyan) [reply data/sql/base/db_characters/arena_team_member.sql]: Done

**Comment by Winfidonarleyan** [comment]
Ready, @Kitzunu can R?

File: data/sql/base/db_characters/quest_tracker.sql [hunk 2]
```diff
@@ -12,7 +12,7 @@
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
--- Дамп структуры для таблица _acore_characters.quest_tracker
+-- Дамп структуры для таблица acore_characters.quest_tracker
 DROP TABLE IF EXISTS `quest_tracker`;
 CREATE TABLE IF NOT EXISTS `quest_tracker` (
   `id` mediumint(8) unsigned NOT NULL DEFAULT 0,
```
Comment (Kitzunu) [inline data/sql/base/db_characters/quest_tracker.sql]: .

File: data/sql/base/db_world/achievement_criteria_data.sql [hunk 3]
```diff
@@ -12,7 +12,7 @@
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
--- Дамп структуры для таблица _acore_world.achievement_criteria_data
+-- Дамп структуры для таблица acore_world.achievement_criteria_data
 DROP TABLE IF EXISTS `achievement_criteria_data`;
 CREATE TABLE IF NOT EXISTS `achievement_criteria_data` (
   `criteria_id` mediumint(9) NOT NULL,
```
Comment (Kitzunu) [inline data/sql/base/db_world/achievement_criteria_data.sql]: .

File: data/sql/base/db_world/achievement_reward.sql [hunk 4]
```diff
@@ -12,7 +12,7 @@
 /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
 
--- Дамп структуры для таблица _acore_world.achievement_reward
+-- Дамп структуры для таблица acore_world.achievement_reward
 DROP TABLE IF EXISTS `achievement_reward`;
 CREATE TABLE IF NOT EXISTS `achievement_reward` (
   `ID` mediumint(8) unsigned NOT NULL DEFAULT 0,
```
Comment (Kitzunu) [inline data/sql/base/db_world/achievement_reward.sql]: .

**Review by Kitzunu — APPROVED** [review]

**Comment by Winfidonarleyan** [comment]
@Kitzunu thanks, tomorrow replace this

**Comment by Winfidonarleyan** [comment]
@Kitzunu done

**Comment by Nyeriah** [comment]
Hey winfi, is there anything in the wiki that needs to be updated? What's the procedure this time? 

We should merge this ASAP, as soon as it's ready imo.

**Comment by Winfidonarleyan** [comment]
> Hey winfi, is there anything in the wiki that needs to be updated? What's the procedure this time?
> 
> We should merge this ASAP, as soon as it's ready imo.

I can't find thjs info in wiki, can link?

**Comment by Kitzunu** [comment]
> > Hey winfi, is there anything in the wiki that needs to be updated? What's the procedure this time?
> > We should merge this ASAP, as soon as it's ready imo.
> 
> I can't find this info in wiki, can link?

https://github.com/azerothcore/wiki/pull/640

**Comment by Winfidonarleyan** [comment]
> > > Hey winfi, is there anything in the wiki that needs to be updated? What's the procedure this time?
> > > We should merge this ASAP, as soon as it's ready imo.
> > 
> > 
> > I can't find this info in wiki, can link?
> 
> [azerothcore/wiki#640](https://github.com/azerothcore/wiki/pull/640)

yes yes

**Comment by Kitzunu** [comment]
@Winfidonarleyan could you just make sure that we don't have FLOAT UNSIGNED anywhere and that those use check constraints? https://www.azerothcore.org/wiki/sql-standards#float-double-decimal

**Comment by Winfidonarleyan** [comment]
> @Winfidonarleyan could you just make sure that we don't have FLOAT UNSIGNED anywhere and that those use check constraints? https://www.azerothcore.org/wiki/sql-standards#float-double-decimal

Yes, I checked, everything is fine.

**Pull request merged**